.gitignore

```diff
@@ -12,6 +12,7 @@
 # Generated files
 build
 /reports
+.kotlin
```

Comment:
This seems to be new, some working files appear in `.kotlin`.

Reply:
ah that's expected: https://kotlinlang.org/docs/whatsnew-eap.html#new-directory-for-kotlin-data-in-gradle-projects